test/TEST-75-RESOLVED/test.sh

```diff
@@ -36,6 +42,9 @@ test_append_files() {
 
     # Install DNS-related utilities (usually found in the bind-utils package)
     image_install delv dig host nslookup
+
+    # Install nftables
+    image_install nft
```

Comment:
hmm, i just realized that this test actually sets TEST_NO_QEMU=1 and i am wondering why...

and what confuses me in particular is that you can do nft from inside a container. I had asumed this wouldn't work. /me is puzzled about this.

Reply:
so apparently nft just works in network namespaces, hence all good. (just checked with some namespacing kernel folks upstream kernel)